src/components/Sep6/Sep6Deposit.tsx

```diff
@@ -13,15 +13,20 @@ import {
 import { useRedux } from "hooks/useRedux";
 import { ActionStatus } from "types/types.d";
 
-export const Sep6Send = () => {
+export const Sep6Deposit = () => {
   const { sep6DepositAsset } = useRedux("sep6DepositAsset");
-  const { depositResponse } = sep6DepositAsset;
+  const {
+    data: { depositResponse },
+  } = sep6DepositAsset;
 
   interface FormData {
     depositType: {
       type: string;
     };
-    fields: {
+    infoFields: {
+      [key: string]: string;
+    };
+    customerFields: {
```

Comment:
this PR focuses on the `withdraw` flow, but I made some changes to the `deposit` flow to make the models for each similar in shape